### DISCUSSION TITLE
Nebula Class

### DISCUSSION THREAD

**Discussion by aecolesuk** [bcholmes/StarTrek2d20#221] [General]
Hi, from a search it looks like the Nebula class used to be available on the STA builder, but I can't find it. Was it removed?
What can I do to get it back or have it created?

**Comment by FortunaeSD** [comment]
Its still in the Starship builds under the standard frames, I just built one to see if it was there and it worked.

**Comment by aecolesuk** [comment]
OK, I tried multiple times and couldn't see it. I just typed out the route I was taking through the tool and its right there.
Sorry for wasting time.
